src/routes/docs/advanced/platform/release-policy/+page.markdoc

```diff
@@ -0,0 +1,155 @@
+
+All the self-hosted versions of Appwrite `>=1.x.x` continue to have support and backward compatibility with the Appwrite API and SDKs within each major version. 
+In case a new version of the products has been released and you decide to update, you should expect your applications to continue working properly without any action from your side.
+
+Once we release a version of the product and you decide to upgrade, look in the changelog to understand if your version requires migration of data from your previous setup. 
+This is usually required when we make adjustments to the under the hood data structure for supporting new features and improving maintainability. 
+If this is the case, you could use our built-in migration tool for helping you to upgrade your self-hosted Appwrite version.
+
+{% arrow_link href="/docs/advanced/self-hosting/update" %}
+Learn more about updating self-hosted Appwrite
+{% /arrow_link %}
+
+# Runtime versioning {% #runtime-versioning %}
+
+Appwrite runtimes are built around a combination of operating system, programming language, and software libraries that are subject to maintenance and security updates.
+Appwrite will support and maintain runtimes as a package, which covers the specific combination of operating system, programming language, and software libraries.
+
+Appwrite will support the latest stable versions of our serverless runtime environments for a minimum of 24 months after its initial release as long as security updates for components of the specific runtime are still provided. 
+You can review the list of supported runtimes on Appwrite.
+
+In most cases, the end-of-life date of a language version or operating system is known well in advance. 
+The links below give end-of-life schedules for each language that Appwrite supports as a managed runtime.  
+
+{% table %}
+* Language
+* LTS policy
+---
+* Node.js
+* [https://nodejs.org](https://nodejs.org)
+---
+* Python
+* [devguide.python.org](https://devguide.python.org)
+---
+* Ruby
+* [www.ruby-lang.org](https://www.ruby-lang.org)
+---
+* Java
+* [www.oracle.com](https://www.oracle.com)
+---
+* .NET Core
+* [dotnet.microsoft.com](https://dotnet.microsoft.com)
+---
+* PHP
+* [https://www.php.net](https://www.php.net)
+---
+* Dart
+* [https://dart.dev/](https://dart.dev/)
+---
+* Deno
+* [https://deno.com/runtime](https://deno.com/runtime)
+---
+* Swift
+* [https://developer.apple.com/swift](https://developer.apple.com/swift)
+---
+* Kotlin
+* [https://kotlinlang.org](https://kotlinlang.org)
+---
+* C++
+* [https://en.cppreference.com/w/](https://en.cppreference.com/w/)
+---
+* Alpine
```

Comment:
Alpine is an OS, so do we want to do a separate table? Some images can use Ubuntu/Debian, and do we want to include Windows versions?